docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1099 @@
+    ```csharp
+    /// callback
+    void speechDetected(object sender, EventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetSpeechDetectedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SpeechDetected += speechDetected;
+
+            /// Unset the callback
+            VoiceControlManagerClient.SpeechDetected -= speechDetected;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the conversation requested callback that is invoked when the voice control manager or engine requests conversation for additional information about the current utterance:
+    ```csharp
+    /// callback
+    void conversationRequested(object sender, ConversationRequestedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetConversationRequestedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.ConversationRequested += conversationRequested;
+
+            /// Unset the callback
+            VoiceControlManagerClient.ConversationRequested -= conversationRequested;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+-   Set the private data updated callback invoked when the voice control engine sets private data to the voice control manager:
+    ```csharp
+    /// callback
+    void privateDataUpdated(object sender, PrivateDataUpdatedEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetPrivateDataUpdatedCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.PrivateDataUpdated += privateDataUpdated;
+
+            /// Unset the callback
+            VoiceControlManagerClient.PrivateDataUpdated -= privateDataUpdated;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the specific engine result callback that is invoked when the voice control engine sends additional information about the undefined data:
+
+    ```csharp
+    /// callback
+    void specificEngineResult(object sender, SpecificEngineResultEventArgs e)
+    {
+        /// Your code
+    }
+
+    void SetUnsetSpecificEngineResultCb()
+    {
+        try
+        {
+            /// Set the callback
+            VoiceControlManagerClient.SpecificEngineResult += specificEngineResult;
+
+            /// Unset the callback
+            VoiceControlManagerClient.SpecificEngineResult -= specificEngineResult;
+        }
+        catch (Exception e)
+        {
+            /// Error handling
+        }
+    }
+    ```
+
+
+-   Set the pre recognition result updated callback invoked when voice control engine sets the pre recognition results (partial ASR) to voice control manager.
+
+    You can get the results of pre result event type and the pre recognition results, the partial ASR.
```

Comment:
Please add a colon instead of a period as it is followed by a code.

You can get the results of pre result event type and the pre recognition results, the partial ASR:

Reply:
@surbhig12 
Thank you your comment. Fixed it.